gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -75,1848 +57,6 @@
 hl._set_flags(no_whole_stage_codegen="1")
 
 
-def add_genotype(mt_path: str, min_hom_threshold: float = 0.95) -> hl.MatrixTable:
```

Comment:
does everything from this file wind up in either /utils/filtering.py or /utils/annotations.py that doesn't stay in place? are there any other places that functions end up

Reply:
yeah, or utils/exports.py